spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -13,17 +26,34 @@ def given_there_is_an_edition
     @edition = create(:edition, document_type: document_type)
   end
 
+  def given_there_is_an_edition_with_a_pre_release_document_type
```

Comment:
Minor: we can reduce the future work to remove the pre-release flag by not mentioning the 'pre-release' aspect, since it's only temporary and tangential to the feature.

Reply:
I'll rename this, but I'll need to keep `pre_ release` set to true in the document type